README.md

```diff
@@ -12,8 +12,10 @@ This project is a spinoff from [FLAML](https://github.com/microsoft/FLAML).
     <img src="https://github.com/microsoft/autogen/blob/main/website/static/img/flaml.svg"  width=200>
     <br>
 </p> -->
-
-:fire: Heads-up: pyautogen v0.2 will switch to using openai v1.
+:fire: Nov 8: AutoGen is selected into [Open100: Top 100 Open Source achievements](https://www.benchcouncil.org/evaluation/opencs/annual.html) 40 days after release.
```

Comment:
I think you might want to double space these or put then in an unordered list to get em each on their own line in the final render. Otherwise seems a simple enough update. Really would look better on their own line tho. (at least "View File" renders them in one big run-on)

Reply:
Yes you are right. I forgot that... again!